openapi3/loader_exmaple_schema_ref_test.go

```diff
@@ -0,0 +1,17 @@
+package openapi3
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestLoadSpecWithExampleRefToSchemas(t *testing.T) {
+	loader := NewLoader()
+	loader.IsExternalRefsAllowed = true
```

Comment:
This does not seem required for this test.

Reply:
yes,removed